atomic-structure/content.md

```diff
@@ -0,0 +1,25 @@
+# Radioactive Decay
+#### Learning objective
```

Comment:
I just thought it looked better on the screen. If they all have to look the same then it's not a big deal to me.